pandas/core/frame.py

```diff
@@ -6972,7 +6972,7 @@ def value_counts(
         normalize : bool, default False
             Return proportions rather than frequencies.
         sort : bool, default True
-            Sort by frequencies.
+            Sort by frequencies when True. Sort by index when False.
```

Comment:
This is incorrect. It sorts the values that are counted

Reply:
@phofl Thanks for your response :). I tested the example below. When sort=True it sorts by frequencies or value counts and when sort=False it sorts by index.


![image](https://user-images.githubusercontent.com/24562835/233861475-769db253-a1ec-4d59-a907-b953cc2974f6.png)

Reply:
But these are the original columns, not
The inddx

Reply:
Fair point. I said index because the series returned by df.value_counts has these columns as index. Let me correct that in the doc string.